docs/.vitepress/theme/index.js

```diff
@@ -15,6 +15,7 @@ import ArcgisCalciteMap from "../../components/mapping/ArcgisCalciteMap.vue";
 import ContentFigure from "../../components/ui/ContentFigure.vue";
 import Sources from "../../components/ui/Sources.vue";
 import Quiz from "../../components/ui/Quiz.vue";
+import HomePageGlobe from "../../components/mapping/HomePageGlobe.vue";
```

Comment:
No need to register this component globally if you directly import it in MyLayout.vue. We likely won't use this component in a markdown file too.